src/utils/amplitude.ts

```diff
@@ -102,3 +103,19 @@ export const logSidevisningSkolepenger = (side: string) => {
     skjemanavn: ESkjemanavn.Skolepenger,
   });
 };
+
+export const logDokumetasjonsbehov = (
+  dokBehov: IDokumentasjon[],
+  skjemanavn: ESkjemanavn
+) => {
+  dokBehov.map((dok) =>
+    logEvent('dokumentasjonsbehov', {
```

Comment:
Hvordan funker "logEvent" hvis man looper igjennom f.eks 10-15 dok.behov? Blir det 15 kall eller klarer den å "throttle" og sende som en batch? (litt usikker på om 15 kall på "send inn"-knappen er en god idé...

Reply:
Hmmm. Det er sant. Jeg sjekker. Redd for at det blir 15 kall nå.

Reply:
Sendes som batch :)